share/spice/tor_node/tor_node.js

```diff
@@ -6,26 +6,101 @@
             return Spice.failed("tor_node");
         }
 
-        // Put together the base of the Spice.
-        var base = get_shared_spice_data(api_result);
-        if (base === null) {
-            return Spice.failed("tor_node");
-        }
+        // Filter out the trigger from the query.
+        var words = DDG.get_query().split(" "),
+            reg   = /tor|onion/i,
+            query = (reg.test(words[0])) ? words.pop() : words.shift();
```

Comment:
This is interesting, should it be updated to use the more "standard approach"? Don't think it makes any difference... only thing I can think of is the triggers changing in future... meaning the `reg` var will need updating (edge case)

```js
        var script = $('[src*="/js/spice/tor_node/"]')[0],
            source = $(script).attr("src"),
            query = decodeURIComponent(source.match(/tor_node\/([^\/]+)/)[1]);
```